_schemaV2.graphql

```diff
@@ -13903,9 +13903,6 @@ type Money {
   minor: Long!
 }
 
-# A money object with amount in minor currency and a currency code, or an error object
-union MoneyOrErrorUnion = Error
```

Comment:
I wonder if this is intentional - digging

Reply:
It looks like this is safe to remove either way if it's not being referenced - this may have been something that should have been excluded all along the same way `DebitCommissionExemptionInput` and the like were excluded.